index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <div class="logo">My Logo</div>
+        <div id="menu-btn">
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+        </div>
+      </nav>
+    </header>
+    <div id="headline-section">
+      <h1>Hey There. I'm Charles.</h1>
+      <div id="introduction-text">I'm a Software Developer</div>
+      <div id="brief-description">
+        I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+      </div>
+      <div id="contact">
+        <div class="contact-mode">
+          <img src="images/github-icon.png" alt="github-icon" width="19.99px" height="20px">
+        </div>
+        <div class="contact-mode">
+          <div class="linkedIn">
+            <div id="linkedIn-i-o">
+              <img src="images/linkedIn_dot.png" alt="linkedIn-I" width="4.97px" height="4.97px">
+              <img src="images/linkedIn_i.png" alt="linkedIn-I" width="4.97px" height="13.72px">
+            </div>
+            <img src="images/linkedIn_n.png" alt="linkedIn-N" width="12.71px" height="16.97px">
+          </div>
+        </div>
+        <div class="contact-mode">
+          <img src="images/angellist.png" alt="angellist" width="14.97px" height="20px">
+        </div>
+        <div class="contact-mode">
+          <img src="images/twitter.png" alt="twitter-icon" width="19.99px" height="19.96px">
+        </div>
+        <div class="contact-mode">
+          <img src="images/medium-icon.png" alt="medium-icon" width="20px" height="16px">
+        </div>
```

Comment:
- [ ] I think it will be better if your list of icons uses a `<ul>` tag to group them, and `<li>` per icon; so that each icon has a link (`<a>` tag).

Apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using `href`.

e.g

```
<ul>
    <li>
         <a href='#">
              <img src="" alt="" />
          </a>
    </li>
</ul>
```